src/material/chips/chips.md

```diff
@@ -1 +1,194 @@
-<!-- TODO: this guide and the related live examples need to be rewritten to reflect the new chips API. -->
+Chips allow users to view and interact with a list of information.
+
+### Static Chips
+
+Chips are always rendered in a container. To create chips, start with a `<mat-chip-set>` element. Then, nest `<mat-chip>` elements inside the `<mat-chip-set>`.
+
+<!-- example(chips-overview) -->
+
+By default, `<mat-chip>` renders a chip with Material Design styles applied. For a chip with no styles applied, use `<mat-basic-chip>`.
+
+<em>Hint: `<mat-basic-chip>` receives the `mat-mdc-basic-chip` CSS class in addition to the `mat-mdc-chip` class. </em>
+
+#### Adding Icons
+
+A chip has two spots to display icons using content projection.
+
+To add an avatar to a chip, nest an element with `matChipAvatar` attribute inside of `<mat-chip>`. You can add an additional icon to the back of a chip by nesting an element with `matChipTrailingIcon` attribute.
+
+See the [accessibility](#accessibility) section for how to create accessible icons.
+
+<!-- example(chips-avatar) -->
+
+#### Disabled appearance
+
+Although `<mat-chip>` is not interactive, you can set the `disabled` Input to give it disabled appearance.
+
+```html
+<mat-chip disabled>Orange</mat-chip>
+```
+
+### Selection Chips
+
+Use `<mat-chip-listbox>` and `<mat-chip-option>` for selecting one or many items from a list. Create a `<mat-chip-listbox>` parent element. If the user may select more than one option, add the `multiple` attribute. Nest a `<mat-chip-option>` element inside the `<mat-chip-listbox>` for each available option.
+
+<!-- example(chips-form-control) -->
+
+#### Disabled `<mat-chip-option>`
+
+Use the `disabled` Input to disable a `<mat-chip-option>`. This  gives the `<mat-chip-option>` a disabled appearance and prevents the user from interacting with it.
+
+```html
+<mat-chip-option disabled>Orange</mat-chip-option>
+```
+
+#### Icons
+
+Like `<mat-chip>`, `<mat-chip-option>` has two spots to render icons via content projection.
+
+To add an avatar to a chip, nest an element with `matChipAvatar` attribute inside of `<mat-chip-option>`.
+
+You can add an additional icon to the back of a chip by nesting an element with either `matChipTrailingIcon` or `matChipRemove` attribute. You can make this icon either static or interactive.
+
+To create a remove button, nest an element with `matChipRemove` attribute inside the `<mat-chip-option>`. The MatChipOption emits the `removed` Output when the user clicks the `matChipRemove`. Be sure to implement the `removed` Output.
+
+See the [accessibility](#accessibility) section for how to create accessible icons.
+
+#### Keyboard Interactions
+
+Users can move through the chips using the arrow keys and select/deselect them with space. Chips also gain focus when clicked, ensuring keyboard navigation starts at the appropriate chip.
+
+### Input chips
```

Comment:
I'm not sure that "input chips" is the correct term here since there's something in the spec that refers to a different use case: https://material.io/components/chips#input-chips. Maybe something like "Chips connected to an input field"?

Reply:
That's a good point. I like "Chips connected to an input field"